spec/run_all_specs.sh

```diff
@@ -0,0 +1,24 @@
+#! /bin/bash
+
+source ../.env
+
+echo "Testing PG"
+
+docker-compose -f docker/docker-compose.pg.yml build spec
+docker-compose -f docker/docker-compose.pg.yml run spec
+
+echo "Testing mysql"
+
+docker-compose -f docker/docker-compose.mysql.yml build spec
+docker-compose -f docker/docker-compose.mysql.yml run spec
+
+echo "Testing sqlite"
+
+docker-compose -f docker/docker-compose.sqlite.yml build spec
+docker-compose -f docker/docker-compose.sqlite.yml run spec
+
+echo "Done testing...stopping/removing images"
+
+docker-compose -f docker/docker-compose.sqlite.yml down
```

Comment:
I think you can remove these if you use the `--rm` flag in the run.

Reply:
I checked this out, `--rm` removes the `spec` container only, and leaves the corresponding db container.  Doing `down` at the end stops the db container, removes the spec and db container and removes the network created for the run.

Reply:
@drujensen We possibly could replace it with a `docker system prune` to `Remove all unused containers, networks, images (both dangling and unreferenced)`

However it could possibly delete stopped containers other than Granite ones.

WDYT?

Reply:
@Blacksmoke16 yeah, I do this all the time.  :+1:

Reply:
I think its fine the way it is.  going to merge this.